src/ucp/tag/rndv.c

```diff
@@ -680,16 +680,45 @@ ucp_rndv_is_rkey_ptr(const ucp_rndv_rts_hdr_t *rndv_rts_hdr, ucp_ep_h ep,
            (ucp_rkey_packed_md_map(rndv_rts_hdr + 1) &
             ep_config->tag.rndv.rkey_ptr_dst_mds) &&
            /* rendezvous mode must not be forced to put/get */
-           (rndv_mode == UCP_RNDV_MODE_AUTO) &&
-           /* need local memory access for data unpack */
-           UCP_MEM_IS_ACCESSIBLE_FROM_CPU(recv_mem_type);
+           (rndv_mode == UCP_RNDV_MODE_AUTO);
```

Comment:
why no need to check for local memory type?

Reply:
> can this API be used for something else in theory?

looks like we don't have a good way to check whether we can access this memory block by the TL or not. we could check only registration capability.
I've returned back the previous behavior.

Reply:
@yosefe, can we assume that rkey_ptr represents XPMEM only?

Reply:
we better not assume such things